cli/src/logger/mod.rs

```diff
@@ -0,0 +1,34 @@
+use chrono::{DateTime, Local, Utc};
+use log::{set_boxed_logger, Level, LevelFilter, Metadata, Record, SetLoggerError};
+
+struct SimpleLogger;
+
+impl log::Log for SimpleLogger {
+  fn enabled(&self, metadata: &Metadata) -> bool {
+    metadata.level() <= Level::Info
+  }
+
+  fn log(&self, record: &Record) {
+    let local_time = Local::now();
+    let utc_time = DateTime::<Utc>::from_utc(local_time.naive_utc(), Utc);
+    if self.enabled(record.metadata()) {
+      if record.level() == Level::Error {
+        eprintln!("{} | {} | {}", utc_time, record.level(), record.args());
+      } else {
+        println!("{} | {} | {}", utc_time, record.level(), record.args());
+      }
+    }
+  }
+
+  fn flush(&self) {}
+}
+
+pub fn init() -> Result<(), SetLoggerError> {
+  let env = std::env::var("ALAN_TECH_ENV").unwrap_or("production".to_string());
+  match env.as_str() {
+    "local" => {
+      set_boxed_logger(Box::new(SimpleLogger)).map(|()| log::set_max_level(LevelFilter::Info))
+    }
+    _ => set_boxed_logger(Box::new(SimpleLogger)).map(|()| log::set_max_level(LevelFilter::Error)), // TODO: update with new logger struct once decide
+  }
```

Comment:
I think it makes sense to have the `match` return the level filter to use and then setting the boxed logger afterwards (also, `|()|` is not necessary)

Reply:
@cdmistman removing `|()|` give me:
```
expected a `std::ops::FnOnce<((),)>` closure, found `()`

expected an `FnOnce<((),)>` closure, found `()`

help: the trait `std::ops::FnOnce<((),)>` is not implemented for `()`
```

Reply:
sorry, i meant changing `|()|` to `||`

Reply:
Did not work. I need an empty tuple: `expected closure that takes a single 0-tuple as argument`. Another option is to create a custom function per each config, something like: 
```rust
fn config_logger_local(_: ()) {
  log::set_max_level(LevelFilter::Info);
}

fn config_logger(_: ()) {
  log::set_max_level(LevelFilter::Error);
}

pub fn init() -> Result<(), SetLoggerError> {
  let env = std::env::var("ALAN_TECH_ENV").unwrap_or("production".to_string());
  match env.as_str() {
    "local" => {
      set_boxed_logger(Box::new(SimpleLogger)).map(config_logger_local)
    }
    _ => set_boxed_logger(Box::new(SimpleLogger)).map(config_logger), // TODO: update with new logger struct once decide
  }
}
```